examples/cli.js

```diff
@@ -32,9 +31,10 @@ async function main() {
     });
 
     process.stdout.write("Google Bard: ");
-    await bot.askStream(res => {
```

Comment:
why askStream changed to ask? is it not working anymore after this pr?

Reply:
Sadly not. I was going to fix that later. I did mention in it in the commit messages.